amplify/backend/function/BorrowItem/src/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "BorrowItem",
+  "version": "2.0.0",
+  "description": "Lambda function generated by Amplify",
+  "main": ",./ts-output/handlers/api/BorrowItem.handler",
```

Comment:
No comma before directory spec. Line should be `"main": "./ts-output/handlers/api/BorrowItem.handler"`